### PR TITLE
Fix: Requirements for form-field-multiple-labels rule from axe-core

### PR DESCRIPTION
**Pull request by vidorteg**
## Pull request checklist

Make sure you:

- [x] Signed the Contributor License Agreement (after creating PR)
- [ ] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: packages/utils-dom/tests/axe.ts [hunk 1]
```diff
@@ -71,6 +93,13 @@ test.serial('aria-hidden-focus', async (t) => {
     });
 });
 
+test.serial('form-field-multiple-labels', async (t) => {
+    await testAxe(t, {
+        incomplete: '<label for="test">Hi</label><label for="test">Foo</label><input type="text" id="test" />',
```
Comment (antross) [inline packages/utils-dom/tests/axe.ts:98]: Generally, an incomplete rule suggests a bug. Is there a reason we expect this case to be incomplete versus explicitly passing or failing?

File: packages/utils-dom/src/htmlelement.ts [hunk 2]
```diff
@@ -260,6 +260,13 @@ export class HTMLElement extends Node {
         return parse5.serialize(this._element as htmlparser2Adapter.Node, { treeAdapter: htmlparser2Adapter });
     }
 
+    /**
+     * https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll
+     */
+    public querySelectorAll(selectors: string): HTMLElement[] {
+        return this.ownerDocument.querySelectorAll(selectors);
```
Comment (antross) [inline packages/utils-dom/src/htmlelement.ts:267]: We need to scope this query to only include results representing nodes under the element it was called on. To do that, I suspect we'll need to call `cssSelect` directly so we can specify the proper scope.

File: packages/utils-dom/src/htmlelement.ts [hunk 3]
```diff
@@ -260,6 +260,13 @@ export class HTMLElement extends Node {
         return parse5.serialize(this._element as htmlparser2Adapter.Node, { treeAdapter: htmlparser2Adapter });
     }
 
+    /**
+     * https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll
+     */
+    public querySelectorAll(selectors: string): HTMLElement[] {
```
Comment (antross) [inline packages/utils-dom/src/htmlelement.ts:266]: While we're here, can we include `querySelector` as well for completeness?

**Review by antross — CHANGES_REQUESTED** [review]

File: packages/utils-dom/src/htmlelement.ts (hunk 2 above)
Comment (vidorteg) [reply packages/utils-dom/src/htmlelement.ts]: good catch, done.

File: packages/utils-dom/tests/axe.ts (hunk 1 above)
Comment (vidorteg) [reply packages/utils-dom/tests/axe.ts]: As axe-core is reporting this as incomplete I created some logic to pass if the right 'incomplete' was reported (per offline discussion)

**Review by antross — APPROVED** [review]

**Pull request merged**